File: python/syntax/7.string.py
# LENGTH OF STRING----------------------------------------
a = "Hello, World!"
print(len(a))  # 13

# CHECK IN STRING----------------------------------------
txt = "The best things in life are free!"
if "free" in txt:
    print("Yes, 'free' is present.")
if "abc" not in txt:
    print("Yes, 'abc' not is present.")

# Splitting a string into array----------------------------------------
txt = "Hello, World!"
print(list(txt))  # ['H', 'e', 'l', 'l', 'o', ',', ' ', 'W', 'o', 'r', 'l', 'd', '!']
# ['H', 'e', 'l', 'l', 'o', ',', ' ', 'W', 'o', 'r', 'l', 'd', '!']

# Joining a list into a string----------------------------------------
arr = ["Hello", "World"]
print(" ".join(arr))  # Hello World

# Slicing a string----------------------------------------
str = "Saransh"
print(str[1])  # a from index 1 only one element
print(str[-2])  # s from last second index only one element
print(str[2:5])  # llo [including:excluding]

# Triming a string----------------------------------------
str = "   Hello, World!   "
print(str.strip())  # "Hello, World!" (removes leading and trailing spaces)
print(str.lstrip())  # "Hello, World!   " (removes leading spaces)
print(str.rstrip())  # "   Hello, World!" (removes trailing spaces)

File: javascript/syntax/7.string.js
// LENGTH OF STRING----------------------------------------
let a = "Hello, World!";
console.log(a.length);  // 13

// CHECK IN STRING----------------------------------------
let txt = "The best things in life are free!";
if (txt.includes("free")) {
    console.log("Yes, 'free' is present.");
}
if (!txt.includes("abc")) {
    console.log("Yes, 'abc' not is present.");
}

// Splitting a string into array----------------------------------------
txt = "Hello, World!";
console.log(Array.from(txt));  // ['H', 'e', 'l', 'l', 'o', ',', ' ', 'W', 'o', 'r', 'l', 'd', '!']

// Joining a list into a string----------------------------------------
let arr = ["Hello", "World"];
console.log(arr.join(" "));  // Hello World

// Slicing a string----------------------------------------
let str = "Saransh";
console.log(str[1]);  // a from index 1 only one element
console.log(str[str.length - 2]);  // s from last second index only one element
console.log(str.substring(2, 5));  // ran [including:excluding] 